docs/README.md

```diff
@@ -338,17 +342,17 @@ boosted-blog
         └── CreatePost.ts
 ```
 
-As we mentioned before, commands are the input of our system. they're requested
-by the users, validate the input, and store one or more events, so we have to
-define two more things:
+As we mentioned before, commands are the input of our system. They're sent
+by the users of our application. When they are received you can validate its data,
+execute some business logic, and register one or more events. Therefore, we have to define two more things:
 
 1. Who is authorized to run this command.
-1. And what events are being triggered when the command is executed.
+1. The events that it will trigger.
```

Comment:
Shouldn't this be like this:

```suggestion
2. The events that it will trigger.
```

Reply:
Oh no no, Markdown only requires `<any number>.` at the beginning of each line and it will render properly